net-im/skypeforlinux/skypeforlinux-1.9.0.2_p1.ebuild

```diff
@@ -9,7 +9,7 @@ inherit eutils rpm
 
 DESCRIPTION="P2P Internet Telephony (VoiceIP) client"
 HOMEPAGE="http://www.skype.com/"
-SRC_URI="https://repo.skype.com/rpm/stable/skypeforlinux_${PV}-1.x86_64.rpm"
+SRC_URI="https://repo.skype.com/rpm/stable/skypeforlinux_${PV/_p/-}.x86_64.rpm"
```

Comment:
Err, the key point was to use `${PN}` instead of `skypeforlinux` since you're using that name now. `-1` was a minor point for the future ;-P.